xgi/algorithms/shortest_path.py

```diff
@@ -5,6 +5,8 @@
 
 from ..utils import utilities
 
+__all__ = ["single_source_shortest_path_length", "shortest_path_length"]
+
```

Comment:
Just checking: did this end up here from another PR? Or is this important to include in this one?